contracts/truefi2/deprecated/LegacyLoanToken2.sol

```diff
@@ -0,0 +1,500 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I think we should copy the on chain version here, instead of the version that we had in the repo once this PR was created.

Reply:
It's not that important. The clue is to have working liquidation and these two are same